microsoft/nni
# Create soft link "_latest" in nni-experiments

### Description ###

A quality-of-life improvement for developers.

It creates/updates a soft link `~/nni-experiments/_latest` pointing to the latest experiment directory.
So you can do something like `cat _latest/log/nni_manager.log` instead of re-typing experiment ID every time.

On Windows [developer mode](https://blogs.windows.com/windowsdeveloper/2016/12/02/symlinks-windows-10/) must be turned on to use symlink. Or otherwise the patch silently does nothing.

### Checklist ###
  - ~~test case~~ no need because it is mostly in a catch block and is harmless on failure
  - ~~doc~~ no need because no one has to care about this

### How to test ###

Create an experiment and check `~/nni-experiments/_latest` points to the experiment directory.
Then create another experiment and check it points to the new directory.

### Release Note ###

Don't mention.
